yarn.lock

```diff
@@ -1668,7 +1668,7 @@
     "@endo/promise-kit" "^1.1.0"
     "@fast-check/ava" "^1.1.5"
 
-"@endo/patterns@^1.3.0":
+"@endo/patterns@^1.1.0", "@endo/patterns@^1.3.0":
```

Comment:
See if this goes away once package.json dependency is fixed.

Reply:
Was indeed stale. Now that it is updated, the `yarn.lock` changes also disappeared. Done.